src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+import { Todo } from '../types/Todo';
+import { User } from '../types/User';
+
+type Props = {
+  users: User[];
+  onAdd: (newTodo: Todo) => void;
+};
+
+export const TodoForm: React.FC<Props> = ({ users, onAdd }) => {
+  const [title, setTitle] = useState<string>('');
+  const [user, setUser] = useState<number>(0);
+  const [submitted, setSubmitted] = useState<boolean>(false);
```

Comment:
it's redundant to specify types for primitives

Reply:
Not fixed 
<img width="837" alt="image" src="https://github.com/user-attachments/assets/be6d5c28-46fb-4484-961e-08fe32a72250">